source/client/gui/screens/StartMenuScreen.cpp

```diff
@@ -499,7 +499,8 @@ void StartMenuScreen::render(int a, int b, float c)
 	glPushMatrix();
 
 	std::string splashText = getSplashString();
-	int textWidth = m_pFont->width(splashText), textHeight = m_pFont->height(splashText);
+    int textWidth = m_pFont->width(splashText);
```

Comment:
Why did you change this?

Reply:
The textHeight variable was unused so I split the single line up into two lines which allowed me to comment textHeight out. The changes I made to it had no affect on the runtime of the program.